### ISSUE TITLE
Why not just use ajax props directly?

### ISSUE THREAD

**Issue by devinivy** [yarn-co/yarn-sdk-behavior#2]
I see a lot of parameter rewriting to the underlying ajax element.  This leads to a bunch of observers recreating the public API of the iron-ajax component.  Is it possible that it will be more maintainable to make the ajax element "public" and not worry about the element itself unnecessarily mirroring the ajax element's properties?  It seems like those properties are only used to forward a value to the underlying iron-ajax element.

**Comment by wswoodruff** [comment]
Yeah, depending on how soon we'd want that upgrade, maybe we could wait until misbehave is working but yeah I agree that mirroring ties up resources with those observers

**Comment by devinivy** [comment]
Barring misbehave, you can also just have people touch `element.ajax.body` (for example) directly.  I notice the `ajax` element appears to be public anyway given that it's not underscore-prefixed.  In yarn-api-user-behavior it uses `requestor` as the property that contains the iron-ajax element, and it's encouraged to use that element directly as much as needed.  If we standardize that property, these behaviors can possibly become more interoperable.
